pkg/web/templates/partials/transaction/transaction_list.html

```diff
@@ -109,10 +109,8 @@
         {{ $created := .Created.Format "January 2, 2006 15:04:05" }}
         <td>{{ $created }}</td>
         {{ if .LastUpdate }}
-        {{ $lastUpdate := .LastUpdate.Format "January 2, 2006 15:04:05" }}
+        {{ $lastUpdate := .LocalizeLastUpdate.Format "January 2, 2006 15:04:05" }}
```

Comment:
In my timezone, I would see `in 4 hours` in the `last update` column after sending an envelope. Localizing the time should resolve this issue.

Reply:
Using the offset appears to work. I took the following screenshot a few minutes after sending a new envelope: https://tinyurl.com/29a7kmh3